pkgs/development/compilers/go/1.21.nix

```diff
@@ -17,8 +17,8 @@
 }:
 
 let
-  useGccGoBootstrap = stdenv.buildPlatform.isMusl || stdenv.buildPlatform.isRiscV;
```

Comment:
cc @zhaofengli

Reply:
Totally missed this. It's nice that we now have a binary bootstrap seed 👍